mass_merge/models/ir_ui_view.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
```

Comment:
What is the need for this file? Just one call to a super method without any extra action, all arguments passed unchanged...